<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGraphicsView" name="mazeGraphicsView"/>
       <widget class="QWidget" name="mazeOptionsWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="generationLabel">
             <property name="text">
              <string>Generation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="generationComboBox">
             <item>
              <property name="text">
               <string>Depth-First Search</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prim's Algorithm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Loop-Erased Random Walk</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="rowsLabel">
               <property name="text">
                <string>Rows</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QSpinBox" name="rowsSpinBox">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="colsLabel">
               <property name="text">
                <string>Columns</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="colsSpinBox">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="generationPushButton">
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="solvingLabel">
             <property name="text">
              <string>Solving</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="solvingCheckBox">
             <property name="text">
              <string>Show Solution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="wallColorPushButton">
             <property name="text">
              <string>Select wall colour</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_1">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="exitPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Maze">
   <property name="text">
    <string>&amp;Import maze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_Maze">
   <property name="text">
    <string>&amp;Export maze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpen_saves_directory">
   <property name="text">
    <string>&amp;Open saves directory</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
